src/Sarif/Baseline/ResultMatching/SarifLogMatcher.cs

```diff
@@ -239,22 +239,28 @@ private Rule GetRuleFromResources(Result result, IDictionary<string, Rule> rules
                 properties = currentRuns.Last().Properties;
             }
 
+            var remappedLogicalLocations = new Dictionary<LogicalLocation, int>();
+
             properties = properties ?? new Dictionary<string, SerializedPropertyInfo>();
 
             List<Result> newRunResults = new List<Result>();
             foreach (MatchedResults resultPair in results)
             {
-                newRunResults.Add(resultPair.CalculateNewBaselineResult(PropertyBagMergeBehavior));
+                Result result = resultPair.CalculateBasedlinedResult(PropertyBagMergeBehavior);
+                newRunResults.Add(resultPair.CalculateBasedlinedResult(PropertyBagMergeBehavior));
```

Comment:
>resultPair.CalculateBasedlinedResult(PropertyBagMergeBehavior) [](start = 34, length = 62)

Is there a reason you didn't pass the temporary `result` you calculated on the previous line?